static/app/__mocks__/api.tsx

```diff
@@ -253,7 +281,7 @@ class Client implements ApiNamespace.Client {
       } else {
         response.callCount++;
         respond(
-          Client.mockAsync,
+          mockResponseOptions.asyncDelay,
```

Comment:
Should these two options be coalesced? We could prefer the mockresponse.asyncDelay and fallback to the client level config.

Reply:
It is coalesced inside the addMockResponse() call.

If we have tests that are generally written with steps: 1. Setup 2. Act 3. Assert then setting `Client.asyncDelay` would happen inside Setup, and this code is running inside Act. 
So we should not read `Client.asyncDelay` here because there's no closure, you'll always get the last value that was set if we read it here.